tools/fastp/fastp.xml

```diff
@@ -0,0 +1,332 @@
+<tool id="fastp" name="fastp" version="0.12.3">
+    <description>Fast all-in-one preprocessing for FastQ files</description>
```

Comment:
([really minor](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-descriptions)!) the description could flow as a sentence naturally from the tool name

Reply:
adjusted this but if you think it could be improved let me know